app/services/metrics/review_turnaround/per_user_project.rb

```diff
@@ -2,6 +2,7 @@ module Metrics
   module ReviewTurnaround
     class PerUserProject < BaseService
       BATCH_SIZE = 500
+      INTERVAL = Time.zone.today.all_day
```

Comment:
I think we should use a method for this. If I remember correctly, Rails will cache this value and it will only change it if you reset the server.

Reply:
Yes! You are right! Thanks to let me know that